data/defaults.yaml

```diff
@@ -337,3 +337,13 @@ prometheus::grok_exporter::package_ensure: 'latest'
 prometheus::grok_exporter::package_name: 'grok_exporter'
 prometheus::grok_exporter::user: 'grok-exporter'
 prometheus::grok_exporter::version: '1.0.0.RC4'
+prometheus::openldap_exporter::download_url_base: 'https://github.com/tomcz/openldap_exporter/releases'
```

Comment:
Instead of defining the values here, can you please directly assign the data in the prometheus::openldap_exporter? That allows puppet-strings to populate the information into the REFERENCE.md